custom/logistics/api.py

```diff
@@ -234,15 +247,13 @@ def sms_user_sync(self, ilsgateway_smsuser, username_part=None, password=None,
                 user.is_active = bool(ilsgateway_smsuser.is_active)
                 user.user_data = user_dict["user_data"]
                 if "phone_numbers" in user_dict:
-                    user.set_default_phone_number(user_dict["phone_numbers"][0])
+                    user.set_default_phone_number(apply_leniency(user_dict["phone_numbers"][0]))
                     try:
-                        user.save_verified_number(self.domain, user_dict["phone_numbers"][0], True,
-                                                  ilsgateway_smsuser.backend)
+                        user.save_verified_number(self.domain, user_dict["phone_numbers"][0], True)
```

Comment:
I think @czue mentioned that we should be setting the backend_id to MOBILE_BACKEND_TEST for each contact by default.